homeassistant/components/device_automation/trigger.py

```diff
@@ -20,28 +24,12 @@
 TRIGGER_SCHEMA = DEVICE_TRIGGER_BASE_SCHEMA.extend({}, extra=vol.ALLOW_EXTRA)
 
 
-class DeviceAutomationTriggerProtocol(Protocol):
+class DeviceAutomationTriggerProtocol(TriggerProtocol, Protocol):
     """Define the format of device_trigger modules.
 
     Each module must define either TRIGGER_SCHEMA or async_validate_trigger_config.
     """
```

Comment:
Should this docstring be adjusted?
```suggestion
    """Define the format of device_trigger modules.

    Extends `TriggerProtocol` in which each module must define either `TRIGGER_SCHEMA` or
    `async_validate_trigger_config`.
    Each module also needs to define `async_get_trigger_capabilities` and `async_get_triggers`
    """
```

Reply:
It's still correct. We could maybe update the docstring to point out TRIGGER_SCHEMA + async_validate_trigger_config are inherited from TriggerProtocol?